Makefile

```diff
@@ -270,7 +270,7 @@ tools: install-kind install-container-structure-test install-helm install-helm-d
 	GOBIN=$(BINDIR) go install honnef.co/go/tools/cmd/staticcheck@latest
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) $(GOLANGCI_LINT_VERSION)
 	# Follow the official documentation to install the `latest` version, because explicitly specifying the version will get an error.
```

Comment:
@llamerada-jp 
According to `git blame` you had wrote this comment. Do you remember what's meant by "explicitly specifying the version will get an error"? Is it safe to change this to specify concrete version?

Reply:
Understood. Thank you.

Reply:
Let's remove this comment. We cannot use the "latest" for a while, and I don't see any reason to avoid specifying hash as long as it works.